core/src/stages/compute_ik.cpp

```diff
@@ -369,13 +385,19 @@ void ComputeIK::compute() {
 		ik_solutions.emplace_back();
 		auto& solution{ ik_solutions.back() };
 		state->copyJointGroupPositions(jmg, solution.joint_positions);
+
+		kinematic_constraints::KinematicConstraintSet kset(robot_model);
+		kset.add(constraints, scene->getTransforms());
+		auto kset_ptr = kset.empty() ? nullptr : &kset;
```

Comment:
The definition of the KinematicConstraintSet should happen _once_ outside the lambda.

Reply:
Thanks